From 3688373eeea018c7b0d240b705c874793b6136cb Mon Sep 17 00:00:00 2001
Message-Id: <3688373eeea018c7b0d240b705c874793b6136cb.1228954349.git.christian@jaeger.mine.nu>
In-Reply-To: <030d49bbacadc03ef604478be7d0582bd2703cf7.1228954349.git.christian@jaeger.mine.nu>
References: <030d49bbacadc03ef604478be7d0582bd2703cf7.1228954349.git.christian@jaeger.mine.nu>
From: Christian Jaeger <christian@jaeger.mine.nu>
Date: Thu, 11 Dec 2008 00:05:24 +0100
Subject: [PATCH] Fix fix: search these without -* appendix.

Still, I don't know whether this is the proper solution. But at least
it works for me.

Signed-off-by: Christian Jaeger <christian@jaeger.mine.nu>
---
 debian/initramfs/cryptroot-hook |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/debian/initramfs/cryptroot-hook b/debian/initramfs/cryptroot-hook
index 1e30b50..263edf4 100644
--- a/debian/initramfs/cryptroot-hook
+++ b/debian/initramfs/cryptroot-hook
@@ -414,7 +414,7 @@ add_crypto_modules() {
 	done
 
 	if [ -z "$found" ]; then
-		for file in $(find "$MODULESDIR/kernel/drivers/" -name "$mod-*.ko"); do
+		for file in $(find "$MODULESDIR/kernel/drivers/" -name "${mod}.ko"); do
 			altmod="${file##*/}"
 			altmod="${altmod%.ko}"
 			manual_add_modules "$altmod"
-- 
1.6.0.5

